products/erlang.md

```diff
@@ -10,6 +10,13 @@ changelogTemplate: https://github.com/erlang/otp/releases/tag/OTP-__LATEST__
 eoasColumn: true
 releaseDateColumn: true
 
+identifiers:
+-   repology: erlang
+-   cpe: cpe:/a:erlang:erlang
+-   cpe: cpe:2.3:a:erlang:erlang
+-   cpe: cpe:/a:erlang:erlangVotp
+-   cpe: cpe:2.3:a:erlang:erlangVotp
```

Comment:
```suggestion
```

This one is odd, is it really the same ?

Reply:
not sure, the CPE doesn’t even exist so i commited your change